Libplanet.Net/BlockCompletion.cs

```diff
@@ -438,7 +438,7 @@ public async Task WaitAll(CancellationToken cancellationToken = default)
                     if (tasks.Any())
                     {
                         var tcs = new TaskCompletionSource<object>();
-                        using CancellationTokenRegistration ctr = cancellationToken.Register(
+                        cancellationToken.Register(
```

Comment:
Declaring `ctr` (even no usage) seemed be intended due to auto disposing. 🤔 (see also: https://github.com/SonarSource/sonar-dotnet/issues/3137)

so I think we may need to keep previous with new comments to suppress lint.
cc @planetarium/libplanet

Reply:
ok thanks. i will check it!

Reply:
As Sonar Analyzer does not provide any way to discard this pattern of usage yet, we should manually suppress the warning at this moment, like:

```csharp
#pragma warning disable S1481  // Though var _ is unused, the using statement is still needed
using CancellationTokenRegistration _ = cancellationToken.Register(
    () => tcs.TrySetCanceled(),
    useSynchronizationContext: false
);
#pragma warning restore S1481
```